src/pudl/metadata/resources/nrelatb.py

```diff
@@ -4,7 +4,7 @@
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {  # }
     "core_nrelatb__yearly_projected_financial_cases": {
-        "description": "Financial assumptions for each model_case_nrelatb and technology_description.",
+        "description": "Financial assumptions for each model_case_nrelatb, model_tax_credit_case_nrelatb and technology_description.",
```

Comment:
This description doesn't make a ton of sense as an outsider, but maybe it would if you knew what you were looking for / were more familiar with ATB?

Reply:
Oops, unclear and incorrect. Updated the ATB table descriptions in general to clarify.